client/extensions/woocommerce/lib/analytics/index.js

```diff
@@ -8,16 +8,9 @@ import debugFactory from 'debug';
  */
 import * as tracks from 'calypso/lib/analytics/tracks';
 import * as tracksUtils from './tracks-utils';
-import { isTestSite } from 'woocommerce/state/sites/setup-choices/selectors';
-import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 const debug = debugFactory( 'woocommerce:analytics' );
 
 export const tracksStore = {
-	isTestSite: function () {
```

Comment:
We're removing this one because:

* It relied on the `isTestSite` selector
* `isTestSite` selector relied on the `setup-choices` data to be there.
* The `setup-choices` actions exist, but were never called.
* Therefore, we can remove the actions, therefore the reducer and the selectors.
* Once we remove the entirety of unused `setup-choices` state tree, the `isTestSite()` method is useless, so its usage can be removed completely, which is what we're doing here.

To conclude, we already perform an additional check on the API side to verify that sites with `dotcom-store-test-site` flag don't record events, so this check wasn't necessary for the front end at all.